tests/testthat/test-opart-gaussian.R

```diff
@@ -28,3 +28,20 @@ res <- opart_gaussian(c(1,2,3,4,5), 1000000)
 test_that("all the data points in one segment", {
   expect_equal(res$end.vec, 5)
 })
+
+#test for neuroblastoma data set with profile.id = 1 and chromosome = 1
+
+selProfile <- read.csv("neuroblastoma.csv", header = TRUE, sep = ",")
+nrows <- nrow(selProfile)
+
+#test for large penalty
+res <- opart_gaussian(selProfile$logratio, 10000000)
+test_that("all the data points in one segment", {
+  expect_equal(res$end.vec, nrows)
+})
+
+#test for zero penalty
```

Comment:
@tdhock , surprisingly for zero penalty, instead of 'n' segments I got (n-2) segments. I verified the cost vector with fpop which was equal and with (n-1) segments.

Reply:
look at the data. are two consecutive data points equal? that could be the reason. that is good that the cost is the same.